code/modules/projectiles/guns/projectile/constructable/railgun.dm

```diff
@@ -177,9 +180,13 @@
 			to_chat(user, "There is already a rod in the barrel of \the [src].")
 			return
 		to_chat(user, "You load a rod into the barrel of \the [src].")
-		var/obj/item/stack/rods/R = W
-		rod_loaded = 1
-		R.use(1)
+		if(istype(W, /obj/item/stack/rods))
+			var/obj/item/stack/rods/R = W
+			R.use(1)
+			rod_loaded = 1
+		else if(istype(W, /obj/item/weapon/nullrod))
+			qdel(W)
+			rod_loaded = "Holy"
```

Comment:
Excuse me but what the fuck is this? How did this shit get merged?

Reply:
>party member of previous game returns as an antagonist in the sequel